cmd/provider/internal/config/delegatedrouting.go

```diff
@@ -64,6 +69,9 @@ func (c *DelegatedRouting) PopulateDefaults() {
 	if c.CidTtl == 0 {
 		c.CidTtl = defaultDelegatedRoutingCidTtl
 	}
+	if c.AdFlushFrequency == 0 {
+		c.AdFlushFrequency = defaultAdFlushFrequency
```

Comment:
should there be a way to not do the time-based flush?

Reply:
Having zero to disable it sounds like a reasonable option. Let me add it.